clustered/client/src/main/java/org/ehcache/clustered/client/internal/ClusterTierManagerClientEntityFactory.java

```diff
@@ -114,14 +115,13 @@ public void create(final String identifier, final ServerSideConfiguration config
     boolean finished = false;
 
     try {
-      EntityRef<InternalClusterTierManagerClientEntity, ClusterTierManagerConfiguration, Void> ref = getEntityRef(identifier);
+      EntityRef<ClusterTierManagerClientEntity, ClusterTierManagerConfiguration, Void> ref = getEntityRef(identifier);
       try {
         while (true) {
           ref.create(new ClusterTierManagerConfiguration(identifier, config));
           try {
-            InternalClusterTierManagerClientEntity entity = ref.fetchEntity(null);
+            ClusterTierManagerClientEntity entity = ref.fetchEntity(null);
             try {
-              entity.setTimeouts(entityTimeouts);
```

Comment:
Without the two calls, I'm wondering if some of the "finished" ugliness can be cleaned up?

Reply:
Yes